### PR TITLE
Add support for `verdepcheck` action

### PR DESCRIPTION
**Pull request by averissimo**
# Pull Request

Part of https://github.com/insightsengineering/coredev-tasks/issues/528

### Changes description

- Bumps `maditr` (`as.data.table` not being detected)
- Bumps `parallelly` due to API requirement (`omit` parameter)

#### Considerations

ℹ️ on `max` failing: 
- `cran/Matrix` max version requires R 4.4 and `max` strategy is failing

#### See

https://github.com/insightsengineering/tern.mmrm/actions/runs/8799136588

image

### CODE REVIEW

File: .github/workflows/scheduled.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Scheduled 🕰️
+
+on:
+  push:
+    branches:
+      - 528-verdepcheck_support@main
```
Comment (averissimo) [inline .github/workflows/scheduled.yaml:7]: Remove before merge

```suggestion
  push:
    branches:
      - 528-verdepcheck_support@main
```

**Comment by vedhav** [comment]
For some weird reason in my min-isolated verdepcheck I get this test failure:
```r
── Failed tests ────────────────────────────────────────────────────────────────
Failure ('test-fit_mmrm.R:106:3'): fit_mmrm works with character ID variable
coef(result$fit) (`actual`) not identical to coef(expected$fit) (`expected`).

     actual    | expected                
 [1] 30.777407 - 30.777475 [1]           
 [2] 1.530595  - 1.530500  [2]           
 [3] 5.643568  - 5.643565  [3]           
 [4] 0.326027  - 0.326062  [4]           
 [5] 3.774415  - 3.774230  [5]           
 [6] 4.839604  - 4.839588  [6]           
 [7] 10.342167 - 10.342113 [7]           
 [8] 15.053786 - 15.053898 [8]           
 [9] -0.042091 - -0.041926 [9]           
[10] -0.693807 - -0.693685 [10]          
 ... ...         ...       and 1 more ...

[ FAIL 1 | WARN 0 | SKIP 40 | PASS 178 ]
Deleting unused snapshots:
• subgroups/mmrm-forest-plot-with-customizations.svg
• subgroups/mmrm-forest-plot.svg
Error: Test failures
Execution halted

1 error ✖ | 0 warnings ✔ | 0 notes ✔
```

**Comment by vedhav** [comment]
I ran it again, and it passed 🤷 So, I think we can merge and monitor the CI for future changes.

Just one small question, we should add `r-lib/generics` to the `Config/Needs/verdepcheck` right?

File: DESCRIPTION [hunk 2]
```diff
@@ -30,33 +30,40 @@ Depends:
...
+    rmarkdown (>= 2.19),
     testthat (>= 3.1),
-    vdiffr,
+    vdiffr (>= 1.0.7),
     withr (>= 2.0.0)
 VignetteBuilder:
     knitr
+Config/Needs/verdepcheck:
+    insightsengineering/tern, mllg/checkmate, wilkelab/cowplot,
+    tidyverse/dplyr, rvlenth/emmeans, insightsengineering/formatters,
```
Comment (averissimo) [inline DESCRIPTION:62]:
```suggestion
    tidyverse/dplyr, rvlenth/emmeans, insightsengineering/formatters, r-lib/generics,
```

**Comment by averissimo** [comment]
Great!! Added that to the section

**Review by vedhav — APPROVED** [review]

**Pull request merged**